ex/get_friends.pl

```diff
@@ -0,0 +1,38 @@
+use Modern::Perl;
+use Facebook::OpenGraph;
+
+# get access_token for application
+my $token_ref = Facebook::OpenGraph->new(
+    +{
+        app_id => 75324055986753,
+        secret => '291d7f26f909c137bba22cdf809c393',
+    }
+)->get_app_token;
+
+use Data::Dumper;
+say Dumper $token_ref;
+say Dumper $token_ref->{access_token};
+
+# user authorization
+my $fb = Facebook::OpenGraph->new(
+    +{
+        app_id       => 35324055986753,
+        secret       => '291d7f26f909c137bba22cdf809c393',
+        namespace    => 'mishin_narod_ru',
+        redirect_uri => 'https://sample.com/auth_callback',
+    }
```

Comment:
Are these your actual app settings?
If so, I believe you shouldn't include them here.
If not, maybe it's a good idea to change this to obvious dummy data or environment variable such as below so you won't surprise anyone:
```perl
Facebook::OPenGraph->new(+{ app_id => $ENV{YOUR_APP_ID, ....});
```